### ISSUE TITLE
Error on installing using yarn.

### ISSUE THREAD

**Issue by dashmug** [progre/tslint-config-airbnb#9]
```
error An unexpected error occurred: "https://registry.yarnpkg.com/tslint-config-airbnb/-/tslint-config-airbnb-5.1.0.tgz: Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?".
```

It may be a problem of `yarn` and not of this project but I'm logging it here regardless.

**Comment by progre** [comment]
I republish now.
It was fixed.

**Comment by progre** [comment]
Thx to your pointing out!
